Common/include/linear_algebra/CPastixWrapper.hpp

```diff
@@ -93,7 +93,7 @@ class CPastixWrapper
    * \brief Run the external solver for the task it is currently setup to execute.
    */
   void Run() {
-    dpastix(&state, MPI_COMM_WORLD, nCols, colptr.data(), rowidx.data(), values.data(),
+    dpastix(&state, SU2_MPI::GetComm(), nCols, colptr.data(), rowidx.data(), values.data(),
```

Comment:
I know it is a bit nit picky but can we rename GetComm() to just Comm(), just to make things shorter.

Reply:
I agree that the proposed changes make the code slightly less readable.

Unfortunately, the name `SU2_MPI::Comm` is already defined in mpi_structure.hpp, line 98, as a `typedef` to `MPI_Comm`. Maybe this would have to be renamed ?

This change would also break consistency with other methods of `SU2_MPI`, such as `GetRank()` and `GetSize()`.

Reply:
Hmmm alright consistency is important, especially since we don't have a lot of it.